pkg/envtest/server.go

```diff
@@ -87,7 +89,7 @@ type Environment struct {
 	// UseExisting indicates that this environments should use an
 	// existing kubeconfig, instead of trying to stand up a new control plane.
 	// This is useful in cases that need aggregated API servers and the like.
-	UseExistingCluster bool
+	UseExistingCluster *bool
```

Comment:
IMO we don't have to use a boolean pointer. And this is actually a breaking change, since you change the type of a public struct.